ConsoleApp/Account.cs

```diff
@@ -0,0 +1,68 @@
+namespace Bank
+{
+    public class Account
+    {
+        private static int _id;
+        private decimal _balance;
+        private Operation[] _operations;
+        
+        private void RegisterOperation(decimal amount, decimal balance)
+        {
+            var operations = _operations;
+            Array.Resize(ref operations, operations.Length + 1);
+            _operations = operations;
+            _operations[^1] = new Operation(++_id, DateTime.Now, amount, balance);
+        }
+        
+        public Account()
+        {
+            _operations = Array.Empty<Operation>();
+        }
+        
+        public void Deposite(decimal amount)
+        {
+            bool isPositive = Validator.IsPositive(amount); 
```

Comment:
not critical, for negative it's better to throw exception so user will understand something wrong happened